docs/migration/4_0.md

```diff
@@ -1,3 +1,28 @@
 # Technical Changes in Spartacus 4.0
 
+### OccConfigLoaderModule
+- `OccConfigLoaderModule` was removed.
```

Comment:
We removed the whole `config-loader` folder. So, all the exported stuff in this folder should be added into `removed-public-api-deprecations` in schematics.